src/app/(auth)/login/page.tsx

```
@@ -22,6 +23,10 @@ export default function Login() {
   const { push } = useRouter();
   const validEmail = (e: string) => e !== '' && isEmail(e);
 
+  useEffect(() => {
+    if (auth && auth.userId) push(CONFIG.settings);
```

nit: if `auth` is not defined, we should throw an error instead